### PR TITLE
AT 03.02.03 | Multiconfiguration Project > Enable Disabled Multiconfiguration Project

### PR DESCRIPTION
**Pull request by LLrvk**
add test EnableDisabledMulticonfigurationProject
with dependsOnMethod

### CODE REVIEW

**Comment by LLrvk** [comment]
[AT 03.02.03 | Multiconfiguration Project > Enable Disabled Multiconfiguration Project](https://trello.com/c/Skb7AqQY/714-at-030203-multiconfiguration-project-enable-disabled-multiconfiguration-project)

**Review by Stanislaw1349 — APPROVED** [review]
LGTM

File: src/test/java/MulticonfigurationProjectTest.java [hunk 1]
```diff
@@ -312,5 +312,13 @@ public void testMultiConfigurationProjectConfigureParams() {
 
         Assert.assertEquals(actualDescText,multiConfProjectDescriptionText);
     }
+    @Test(dependsOnMethods = "testCreateMultiConfigurationProjectDisabled")
+    public void testEnableDisabledMultiConfigurationProject(){
+        getDriver().findElement(By.xpath("//span[text()='" + PROJECT_NAME + "']")).click();
+        getDriver().findElement(By.xpath("//*[@id='yui-gen1-button']")).click();
+
+        Assert.assertEquals(getDriver().findElement(By.xpath("//*[@id='yui-gen1-button']")).getText(),
+                "Disable Project");
```
Comment (Stanislaw1349) [inline src/test/java/MulticonfigurationProjectTest.java:321]: xpath желательно без (*)

**Review by judmi — APPROVED** [review]
OK

**Pull request merged**